### PR TITLE
Disabling this test for now due to PHP7 issues

### PR DESCRIPTION
**Pull request by DeepDiver1975**
refs #16643

### CODE REVIEW

**Comment by Xenopathic** [comment]
Please use `$this->markTestSkipped()` instead of changing the test function name, so we get regular reminders to fix this. Broken function name tests get missed: #17993

**Comment by Xenopathic** [comment]
Or in fact use the `@requires PHP` annotation: https://phpunit.de/manual/current/en/incomplete-and-skipped-tests.html#incomplete-and-skipped-tests.skipping-tests-using-requires (this is cool, didn't know about it)

**Comment by DeepDiver1975** [comment]
> Or in fact use the @requires PHP annotation:

doesn't help because you can only specify a min version - not a max version

File: tests/lib/share/searchresultsorter.php [hunk 1]
```diff
@@ -42,6 +42,9 @@ public function testSort() {
      * @expectedException PHPUnit_Framework_Error
      */
 	public function testSortWrongLog() {
+		if (version_compare(PHP_VERSION, '7.0.0', '<')) {
```
Comment (Xenopathic) [inline tests/lib/share/searchresultsorter.php]: Missing a not (`!`) here, or swap the operator for `>=`

**Comment by DeepDiver1975** [comment]
> Please use $this->markTestSkipped() instead of changing the test function name, so we get regular reminders to fix this. Broken function name tests get missed: #17993

aha - I'll comment out the test - this is a pain ...

````
Test_Share_Search::testSortWrongLog
Failed asserting that exception of type "PHPUnit_Framework_SkippedTestError" matches expected exception "PHPUnit_Framework_Error". Message was: "" at
#0 /var/jenkins/workspace/server-master-linux-php7/database/sqlite/label/SLAVE/tests/lib/share/searchresultsorter.php(46): PHPUnit_Framework_Assert::markTestSkipped()
#1 [internal function]: Test_Share_Search->testSortWrongLog()
#2 phar:///usr/local/bin/phpunit/phpunit/Framework/TestCase.php(865): ReflectionMethod->invokeArgs(Object(Test_Share_Search), Array)
#3 phar:///usr/local/bin/phpunit/phpunit/Framework/TestCase.php(743): PHPUnit_Framework_TestCase->runTest()
#4 phar:///usr/local/bin/phpunit/php-invoker/Invoker.php(93): PHPUnit_Framework_TestCase->runBare()
#5 phar:///usr/local/bin/phpunit/phpunit/Framework/TestResult.php(607): PHP_Invoker->invoke(Array, Array, 900)
#6 phar:///usr/local/bin/phpunit/phpunit/Framework/TestCase.php(699): PHPUnit_Framework_TestResult->run(Object(Test_Share_Search))
#7 phar:///usr/local/bin/phpunit/phpunit/Framework/TestSuite.php(722): PHPUnit_Framework_TestCase->run(Object(PHPUnit_Framework_TestResult))
#8 phar:///usr/local/bin/phpunit/phpunit/Framework/TestSuite.php(722): PHPUnit_Framework_TestSuite->run(Object(PHPUnit_Framework_TestResult))
#9 phar:///usr/local/bin/phpunit/phpunit/TextUI/TestRunner.php(406): PHPUnit_Framework_TestSuite->run(Object(PHPUnit_Framework_TestResult))
#10 phar:///usr/local/bin/phpunit/phpunit/TextUI/Command.php(153): PHPUnit_TextUI_TestRunner->doRun(Object(PHPUnit_Framework_TestSuite), Array)
#11 phar:///usr/local/bin/phpunit/phpunit/TextUI/Command.php(105): PHPUnit_TextUI_Command->run(Array, true)
#12 /usr/local/bin/phpunit(537): PHPUnit_TextUI_Command::main()
#13 {main}.

**Comment by Xenopathic** [comment]
Ah, the expected exception. Forgot about that.

**Comment by MorrisJobke** [comment]
Why not simply kill this test? It doesn't add any useful test coverage except that PHP behaves like it behaves -.-

**Comment by LukasReschke** [comment]
+1 for killing it or disabling. Whatever :speak_no_evil: :+1:

**Comment by DeepDiver1975** [comment]
here we go https://github.com/owncloud/core/pull/18052

**Pull request closed without merging**